client/css/global.scss

```diff
@@ -0,0 +1,87 @@
+$overlay-index: 10;
+
+.fm-overlay {
+	transition: opacity .5s ease-in-out;
+	opacity: 0;
+
+	&.visible {
+		opacity: 1;
+	}
+
+}
+
+.fm-overlay-fade {
+	position: fixed;
+	top: 0;
+	left: 0;
+	height: 100%;
+	width: 100%;
+	z-index: $overlay-index;
```

Comment:
The `fixed` may create a new stacking context, in which case this less an issue, but I would use a much higher number, regardless. `5000` or more. It's a good practice to leave lots of room between different stacking contexts so that it's easy to keep them distinct.